pkg/apis/serving/v1/service_validation.go

```diff
@@ -31,8 +31,8 @@ func (s *Service) Validate(ctx context.Context) (errs *apis.FieldError) {
 	// spec validation.
 	if !apis.IsInStatusUpdate(ctx) {
 		errs = errs.Also(serving.ValidateObjectMetadata(ctx, s.GetObjectMeta(), false))
-		errs = errs.Also(serving.ValidateRolloutDurationAnnotation(
-			s.GetAnnotations()).ViaField("annotations"))
+		errs = errs.Also(serving.ValidateRolloutDurationAnnotation(s.GetAnnotations()).ViaField("annotations"))
+		errs = errs.Also(serving.ValidateProgressDeadlineAnnotation(s.GetAnnotations()).ViaField("annotations"))
```

Comment:
I think we should not have users put this on the Service but the Revision template

ie.

```yaml
apiVersion: serving.knative.dev/v1
kind: Service
metadata:
  name: helloworld-go
  namespace: default
spec:
  template:
    metadata:
      annotations:
        serving.knative.dev/progress-deadline: 10s
    spec:
      containers:
        - image: gcr.io/knative-samples/helloworld-go
```

Reply:
Yeah, that should've gone in revision_validation.go ...